packages/webpack/src/utils/with-web.ts

```diff
@@ -432,6 +475,26 @@ function getCommonLoadersForGlobalCss(
   ];
 }
 
+function getCommonLoadersForGlobalStyle(
```

Comment:
Keep this separate from `getCommonLoadersForGlobalCss`, because there are minor differences and we want to avoid bad abstractions like we did previously.